ONSdigital/ras-frontstage
# Share a survey improvement

# What and why?
As part of the ongoing user help journey improvement works, the share a survey page needed to be updated to reflect the desired appearance. This PR achieves this.

# How to test?
1. Run local tests
2. Deploy this PR to your dev env and run acceptance tests
3. Setup a CE in rOps and grab the enrolment code
4. Use enrolment code to add a new survey in Frontstage
5. Click 'Get help with this survey' -> select 'Help with my account' -> Continue -> select 'Share access to my surveys' -> continue -> continue - > select the company - > continue
6. Check the 'Share access to surveys' page looks like the Figma design
7. Check all functionality works and that you can share a survey with a new user

# Jira
https://jira.ons.gov.uk/browse/RAS-1199
